cmake/find/poco.cmake

```diff
@@ -0,0 +1,10 @@
+option (USE_INTERNAL_POCO_LIBRARY "Use internal Poco library" ON)
```

Comment:
Looks like it is better to add a comment why it does not use `NOT_UNBUNDLED` as a default value (otherwise someone like me, may make an attempt to change this).

Something like

```cmake
# unbundled poco has too much issues, and it is not recommended to use it even for unbundled build
# Here is the list of what may not work with unbundled poco:
# - s3 support will not work (since it requires modified Poco::Net::HTTPClientSession::setProxy())
# - custom http headers will not work
# - some build may fails (due to some patches were not accepted into upstream)
```